src/main/flight/pid.c

```diff
@@ -1100,9 +1100,11 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         pidRuntime.oldSetpointCorrection[axis] = setpointCorrection;
 #endif
 
-        // Only enable feedforward for rate mode and if launch control is inactive
-        const float feedforwardGain = (flightModeFlags || launchControlActive) ? 0.0f : pidRuntime.pidCoefficient[axis].Kf;
+        // no feedforward in launch control
+        float feedforwardGain = launchControlActive ? 0.0f : pidRuntime.pidCoefficient[axis].Kf;
         if (feedforwardGain > 0) {
+            // halve feedforward in Level mode since stick sensitivity is weaker by about half
+            feedforwardGain *= (flightModeFlags == ANGLE_MODE) ? 0.5f : 1.0f;
```

Comment:
You could change
```cpp
(flightModeFlags == ANGLE_MODE)
```
to
```cpp
FLIGHT_MODE(ANGLE_MODE)
```
to prevent bugs in case `flightModeFlags` has `ANGLE_MODE` enabled, but isn't equal to `ANGLE_MODE`.
(Don't know if this ever happens in practice but in theory it can cause a bug)

Reply:
Good catch!